common/test/services/prismic/pages.test.js

```diff
@@ -1,7 +1,32 @@
 import { parsePage } from '../../../services/prismic/pages';
 import { pageWithoutBody } from '../../fixtures/pages/page';
+import { whatsOn } from '../../fixtures/pages/whats-on';
+import WhatsOnPage from '../../../../content/webapp/pages/whats-on';
+import { mountWithThemeAndContexts } from '../../../../common/test/fixtures/enzyme-helpers';
 
 describe('pages', () => {
+  describe('whats-on', () => {
+    it('renders different markup with/without a featured exhibition', () => {
```

Comment:
test name could be ( should render no results if there are no featured exhibitions, should render featured exhibitions on whats-on) which we further check other bits if we wanted etc.. (e.g if any li elements exists based on fixture data etc..)